treeherder/perf/migrations/0013_add_alert_timestamps.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2019-02-22 10:32
```

Comment:
Could you regenerate this using Django 2.0, just in case it's changed? (On `master`)